config/config.yaml.dist

```diff
@@ -1,3 +1,11 @@
-PrimeURL: "ws://127.0.0.1:8547"
-RegionURLs: ["ws://127.0.0.1:8579", "ws://127.0.0.1:8581", "ws://127.0.0.1:8583"]
-ZoneURLs: [["ws://127.0.0.1:8611", "ws://127.0.0.1:8643", "ws://127.0.0.1:8675"], ["ws://127.0.0.1:8613", "ws://127.0.0.1:8645", "ws://127.0.0.1:8677"], ["ws://127.0.0.1:8615", "ws://127.0.0.1:8647", "ws://127.0.0.1:8679"] ]
\ No newline at end of file
+# Config for node URLs
+Location: []
+Auto: false
```

Comment:
Please only add viable options to the config. e.g. `Auto`, `Mine`, `Optimize`, `OptimizeTimer`, etc have no affect on CPU miner behavior. These and other such options shouldn't be included in this list

Reply:
Correct, updated in the new version.